src/routes/sensorthings.js

```diff
@@ -0,0 +1,50 @@
+import proxy                   from 'express-http-proxy';
+
+import config                  from '../config';
+import { resourceEndpoints }   from '../routes/base';
+import sensorthings            from 'sensorthings'
+
+const localDb = config.get('sensorthings.local.db');
+let sensorthingsAPI;
+
+if (localDb) {
+  //Init DB
+  sensorthingsAPI = sensorthings;
+} else {
+  /**
+   * This is just a temporary utility.
+   *
+   * This proxy allows us to quickly expose a SensorThings API while we work on
+   * our own open source implementation of this API. We will be using SensorUp's
+   * sandbox.
+   *
+   * You will need to get an access token by creating an account at
+   * https://pg.sensorup.com/playground.html
+   *
+   * Add the access token to your 'sandboxToken' config file entry.
+   */
+  sensorthingsAPI = proxy(config.get('sensorthings.remote.server'), {
+    filter: (req, res) => {
+      const matches = resourceEndpoints.filter(endpoint => {
+        const regexp = new RegExp('^((?!' + endpoint + ').)*$');
+        return req.path.match(regexp) == null;
+      });
+
+      // Only the paths listed in the resourcesEndpoints array are handled
+      // by the proxy.
+      if (matches.length) {
+        return true;
+      }
+    },
+    decorateRequest: (proxyReq, originalReq) => {
+      // Add SensorUp auth header.
+      proxyReq.headers['St-P-Access-Token'] = config.get('sensorthings.remote.credentials.token');
```

Comment:
This header is specific to SensorUp. Maybe we need to add a config option for this. Let's keep it simple for now and assume that an external ST service only takes its credentials through a header:
```json
credentials: {
  header: 'St-P-Access-Token',
  value: 'token'
}
```